app/project_tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+pipenv install --dev
```

Comment:
this should be running in the container so it should already have a virtual environment and the dependencies installed

Reply:
Ah, `${pipenv_dev_arg}` is taking care of it!